### DISCUSSION TITLE
Add link to GitLab profile in the Profile Online presence section

### DISCUSSION THREAD

**Discussion by heliocola** [joemasilotti/railsdevs.com#891] [Ideas]
I would like to propose to add a link to the GitLab account in the Dev Profile.
I started using it more often recently because I had a customer migrating over to GitLab.

Is there anybody else here that also uses GitLab on a regular basis?

**Comment by heliocola** [comment]
I already have a proof of concept in the making and I can push a draft PR, if we want to move forward with it.
The screenshot below is from my localhost.

<img width="380" alt="Screenshot 2024-01-19 at 3 26 18 AM" src="https://github.com/joemasilotti/railsdevs.com/assets/1107956/24694b10-d516-4621-93a8-7ca3d94c22e2">

**Reply by joemasilotti** [reply]
PRs welcome!

**Reply by heliocola** [reply]
Sure thing!
I was about to open PR, and realized maybe I should create an issue first and ask.
I was about to open an issue, and noticed new ideas should be here! :-D

**Reply by joemasilotti** [reply]
Merged! Thanks @heliocola.
